### DISCUSSION TITLE
List titles cannot include "🅰️" emoji

### DISCUSSION THREAD

**Discussion by Casiyre** [community/community#28006] [Mobile] [answered]
When creating a new list on the Android GitHub app, you cannot use the 🅰️ emoji. It just says "No network connection," which isn't true.

This probably happens because it's a weird emoji lol.

**Comment by stevepopovich** [comment, marked as answer]
Hi @Casiyre, nice catch! It turns out that List titles can not include 4-byte Unicode characters
<img width="474" alt="image" src="https://user-images.githubusercontent.com/3252375/182870305-571092ea-fa64-422b-9bf4-d459236837e6.png">

So the Android app is showing the wrong error, which we can fix! Thanks for reporting.

**Reply by stevepopovich** [reply]
This will be fixed in the next 2 releases!
